components/registry/org.wso2.carbon.registry.activities.ui/src/main/resources/web/activities/activity-ajaxprocessor.jsp

```diff
@@ -41,6 +41,19 @@
         ActivityBean activityBean;
         String requestedPage = request.getParameter(UIConstants.REQUESTED_PAGE);
 
+        String sortOrder = request.getParameter("sortOrder");
+        String imgType = "";
+        String displayStr = "display:none;";
+
+        if (sortOrder!=null){
+            if (sortOrder.equals("DES")){
```

Comment:
sortOrder.equals("DES") can be replaced by,
"DES".equals(sortOrder) and eliminate the usage of null check in line 48.

Same solution can be used for below line 51.
After that re-think weather that null check is necessary for line 54 code.